### PR TITLE
[15.0][MIG]account_payment_order_return:Migration to 15.0

### PR DESCRIPTION
**Pull request by bizzappdev**

### CODE REVIEW

**Review by almumu — COMMENTED** [review]
test-requiremenst are now replaced and merged in https://github.com/OCA/account-payment/pull/610

[(https://github.com/oca/account-payment/pull/536#issuecomment-1475448729)]

**Comment by bizzappdev** [comment]
> test-requiremenst are now replaced and merged in [OCA/account-payment#610](https://github.com/OCA/account-payment/pull/610)
> 
> [(https://github.com/[OCA/account-payment/pull/536](https://github.com/OCA/account-payment/pull/536)#issuecomment-1475448729)]
@almumu 
Done

File: account_payment_order_return/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Copyright 2017 Tecnativa - Luis M. Ontalba
+# Copyright 2021 Tecnativa - João Marques
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+{
+    "name": "Account Payment Order Return",
+    "version": "15.0.0.1.0",
```
Comment (pedrobaeza) [inline account_payment_order_return/__manifest__.py]:
```suggestion
    "version": "15.0.1.0.0",
```

File: account_payment_order_return/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    "category": "Banking addons",
+    "author": "Tecnativa, " "Odoo Community Association (OCA)",
```
Comment (pedrobaeza) [inline account_payment_order_return/__manifest__.py]:
```suggestion
    "author": "Tecnativa, Odoo Community Association (OCA)",
```

Comment (bizzappdev) [reply account_payment_order_return/__manifest__.py]: done

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**